portfolio/src/main/webapp/script.js

```diff
@@ -32,8 +32,38 @@ function addRandomQuote() {
 }
 
 //use fetch to request content from server and add it to page.
-async function getMyName() {
-  const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('name-container').innerText = quote;
+// async function getMyName() {
```

Comment:
You can probably remove this commented-out code now? If you think you'll need it again, GitHub stores the file history, so you can always get it from there if you need it.